db/seeds.rb

```diff
@@ -18,6 +18,10 @@
 #   Movie.create(title: title, overview: overview, poster_url: poster_url, rating: rating)
 # end
 
+Offer.destroy_all
+User.destroy_all
+Booking.destroy_all
```

Comment:
Bookings should be destroyed first, before users and offers. 
Deleting a user or offer linked to an existing booking will fail validation.